src/layer/const.ts

```diff
@@ -226,3 +227,10 @@ export enum DEMInstanceType {
   COPERNICUS_30 = 'COPERNICUS_30',
   COPERNICUS_90 = 'COPERNICUS_90',
 }
+
+export enum DEMInstanceTypeOrthorectification {
```

Comment:
This looks as duplication but I assume there is some good reason for this. fftr

Reply:
The reason why this was added as separate variable is because for the orthorectification the copernicus dem with 10 meter resolution can be used but it's not available as "normal" dem instance.